lib/solidus_stripe/testing_support/factories.rb

```diff
@@ -15,6 +15,6 @@
 
   factory :stripe_payment_source, class: 'SolidusStripe::PaymentSource' do
     association :payment_method, factory: :stripe_payment_method
-    stripe_payment_method_id { 'pm_123' }
+    stripe_payment_method_id { "pm_#{SecureRandom.uuid.delete('-')}" }
```

Comment:
This was needed to avoid false positive specs. If all the generated ids have the same value, and we run checks against them, it's very likely that we could target the right object and struggle to realize that.